block/block.go

```diff
@@ -21,7 +22,9 @@ func (m *Manager) applyBlock(block *types.Block, commit *types.Commit, blockMeta
 		return types.ErrInvalidBlockHeight
 	}
 
-	m.logger.Debug("Applying block", "height", block.Header.Height, "source", blockMetaData.source)
+	types.LastAppliedBlockSource.With(prometheus.Labels{"source": blockMetaData.source.String()}).Set(0)
+
```

Comment:
imo should extract a function to metrics.go which takes a source as arg and handes 
```suggestion
		types.LastAppliedBlockSource.WithLabelValues(blockMetaData.source.String()).Set(0)
```